packages/fetchye/rollup.config.js

```diff
@@ -0,0 +1,66 @@
+import { nodeResolve } from '@rollup/plugin-node-resolve';
+import commonjs from '@rollup/plugin-commonjs';
+import babel from '@rollup/plugin-babel';
+import { terser } from 'rollup-plugin-terser';
+import pkg from './package.json';
+
+export default [
+  // browser-friendly UMD build
+  {
+    input: 'src/index.js',
+    output: {
+      name: 'fetchye',
+      file: pkg.browser,
+      format: 'umd',
+    },
+    external: ['react', 'react-dom'],
+    plugins: [
+      nodeResolve({ extensions: ['.js', '.jsx'], browser: true }),
+      commonjs(),
+      babel({
+        babelHelpers: 'bundled',
+        exclude: 'node_modules/**',
+        presets: ['amex'],
+      }),
+      terser(),
+    ],
+  },
+  {
+    input: 'src/index.js',
+    output: {
+      file: pkg.main,
+      format: 'cjs',
+    },
+    external: ['crypto', 'react', 'react-dom'],
```

Comment:
crypto as an external ?

Reply:
@JAdshead I worked with @Francois-Esquire on this, it is because of `object-hash`

Reply:
but why should it be an external if its required for fetchye to work